packages/prosemirror-lwdita/src/tests/schema.spec.ts

```diff
@@ -129,7 +129,8 @@ describe('Function schema()', () => {
     // nodeNames are strings in the content array
     const nodeNames = nodes.content.filter((node: NodeSpec) => typeof node === 'string')
     const expectedNodes = [
-      'text',         'image',       'data',
+      'text',         'hard_break',
```

Comment:
1. This introduces a new "custom" element to the XDITA document, which is non-valid LwDITA: `<hard-break/>` doesn't exist. The correct `<br>` tag has to be transformed into CR LF characters `\r\n` or `&#xA; `, `&#xD;`

Reply:
> This introduces a new "custom" element to the XDITA document
Not in this case, It's being added to the prosemirror document that's all.
> The correct <br> tag has to be transformed into CR LF characters
Agreed.